utensor_cgen/_pbparser_impl.py

```diff
@@ -149,9 +154,35 @@ def _is_freeze_graph(graph_def):
   return is_frozen
 
 
-def _parse_graph_def(graph_def):
+def _parse_graph_nodes_bfs(graph_def, output_nodes=None):
+  if output_nodes is None:
```

Comment:
Should output nodes be required?

Reply:
I think it would be nice if the cli can figure out the output nodes for me or I can overwrite it if necessary.
That's why I make it optional here.

Reply:
cool, I just worry about improperly frozen graphs having the training cost functions etc.

Reply:
Well, it's users' responsibility if they do not provide me the list of output nodes.